### DISCUSSION TITLE
Backup / Restore procedure

### DISCUSSION THREAD

**Discussion by ggoncas** [netbox-community/netbox#8199] [Q&A]
Hi guys, there is any procedure for Netbox backup and restore?. What I should take in mind if I want to restore my current Netbox instance in other Server?

Thanks

**Comment by jk-onnet** [comment]
https://netbox.readthedocs.io/en/stable/administration/replicating-netbox/
You might need to copy your secret key (current NetBox instance) to your other server.

**Reply by candlerb** [reply]
The secret key doesn't do anything except encrypt cookies - if it changes, all that happens is everyone has to login again.
